backend/cli.py

```diff
@@ -57,8 +65,27 @@ def hash_password_cmd() -> None:
 async def create_dev_db() -> None:
     await sql_create_dev_db()
 
+@click.command()
+@click.option('--email', prompt='Email', help='The email used to log into the account.')
+@click.option('--password', prompt='Password', help='The password used to log into the account.')
+@click.option('--name', prompt='Name', help='The name associated with the account.')
+@run_async
+async def register_user(email, password, name) -> None:
```

Comment:
There are types missing here: https://github.com/robigan/bracket/actions/runs/7989651896/job/21816653345
```suggestion
async def register_user(email: str, password: str, name: str) -> None:
```